dokan/dokan.h

```diff
@@ -636,12 +650,10 @@ typedef struct _DOKAN_OPERATIONS {
   *
   * Retrieve all NTFS Streams informations on the file.
   * This is only called if \ref DOKAN_OPTION_ALT_STREAM is enabled.
-  *
-  * Supported since 0.8.0. You must specify the version at
-  * DOKAN_OPTIONS.Version.
-  *
+  * 
+  * \since Supported since version 0.8.0. You must specify the version in \ref DOKAN_OPTIONS#Version.
   * \param FileName File path requested by the Kernel on the FileSystem.
-  * \param FillFindStreamData Callback that has to be called with PWIN32_FIND_STREAM_DATA that contain stream information.
+  * \param FillFindStreamData Callback that has to be called with \ref PWIN32_FIND_STREAM_DATA that contain stream information.
```

Comment:
No ref possible here. PWIN32_FIND_STREAM_DATA is not define in dokan but windows headers.